CONTRIBUTORS.md

```diff
@@ -685,6 +686,7 @@ GitHub contributors:
 - sadlyblue
 - sagaradonis
 - sallemiines
+- Sam
```

Comment:
Is it juraj1000 ?

Reply:
Nothing indicates that it's the same person

Reply:
You'll see it's mail in the Changelog project PR